extensions/amp-a4a/amp-a4a-format.md

```diff
@@ -0,0 +1,358 @@
+
+Missing context class `.amp-animate`.
+```css
+.box {
+    transform: rotate(180deg);
+    transition: transform 2s;
+}
+```
+
+##### Transitionable and animatable properties
+
+The only properties that may be transitioned are opacity and transform.
+([Rationale](http://www.html5rocks.com/en/tutorials/speed/high-performance-animations/))
+
+**Good**
+```css
+transition: transform 2s;
+```
+
+**Bad**
+```css
+transition: background-color 2s;
+```
+
+**Good**
+```css
+@keyframes turn {
+  from {
+    transform: rotate(180deg);
+  }
+  
+  to {
+    transform: rotate(90deg);
+  }
+}
+```
+
+**Bad**
+```css
+@keyframes slidein {
+  from {
+    margin-left:100%;
+    width:300%
+  }
+  
+  to {
+    margin-left:0%;
+    width:100%;
+  }
+}
+```
+
+
+### AMP Extensions
+
+The following are _allowed_ AMP extension modules in an A4A creative. Extensions
+not explicitly allowed are prohibited.
+
+Most of the omissions are either for performance or to make A4A creatives 
+simpler to analyze.
+
+<table>
+  <tr><td>amp-accordion</td></tr>
+  <tr><td>amp-analytics</td></tr>
+  <tr><td>amp-anim</td></tr>
+  <tr><td>amp-audio</td></tr>
+  <tr><td>amp-brid-player</td></tr>
+  <tr><td>amp-brightcove</td></tr>
+  <tr><td>amp-carousel</td></tr>
+  <tr><td>amp-dailymotion</td></tr>
+  <tr><td>amp-facebook</td></tr>
+  <tr><td>amp-fit-text</td></tr>
+  <tr><td>amp-font</td></tr>
+  <tr><td>amp-form</td></tr>
+  <tr><td>amp-fx-flying-carpet</td></tr>
+  <tr><td>amp-google-vrview-image</td></tr>
+  <tr><td>amp-image-lightbox</td></tr>
+  <tr><td>amp-instagram</td></tr>
+  <tr><td>amp-jwplayer</td></tr>
+  <tr><td>amp-kaltura-player</td></tr>
+  <tr><td>amp-lightbox</td></tr>
+  <tr><td>amp-list</td></tr>
+  <tr><td>amp-live-list</td></tr>
+  <tr><td>amp-o2-player</td></tr>
+  <tr><td>amp-pinterest</td></tr>
+  <tr><td>amp-reach-player</td></tr>
+  <tr><td>amp-share-tracking</td></tr>
+  <tr><td>amp-slides</td></tr>
+  <tr><td>amp-social-share</td></tr>
+  <tr><td>amp-soundcloud</td></tr>
+  <tr><td>amp-springboard-player</td></tr>
+  <tr><td>amp-sticky-ad</td></tr>
+  <tr><td>amp-twitter</td></tr>
+  <tr><td>amp-vimeo</td></tr>
+  <tr><td>amp-vine</td></tr>
+  <tr><td>amp-youtube</td></tr>
+</table>
+
+### HTML Tags
+
+The following are _allowed_ tags in an A4A creative.  Tags not explicitly 
+allowed are prohibited.  This list is a subset of the general [AMP tag 
+addendum whitelist](../../spec/amp-tag-addendum.md). Like that list, it is 
+ordered consistent with HTML5 spec in section 4 [The Elements of HTML](http://www.w3.org/TR/html5/single-page.html#html-elements).
+
+Most of the omissions are either for performance or because the tags are not
+HTML5 standard.  For example, `<noscript>` is omitted because A4A depends on
+JavaScript being enabled, so a `<noscript>` block will never execute and,
+therefore, will only bloat the creative and cost bandwidth and latency. 
+Similarly, `<acronym>`, `<big>`, et al. are prohibited because they are not
+HTML5 compatible.
+
```

Comment:
The list of tags that are not allowed is effectively infinite. `<foobar>` is not allowed for example. It might be possible to enumerate the tags listed in the HTML5 spec which are not allowed, for example.